tests/lib/metadata_linux_bridge_test.py

```diff
@@ -0,0 +1,365 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
```

Comment:
This needs to be 2020

Reply:
The commit is from December, the year is good.

Reply:
Ugh, this needs to be automated. GitHub writes `Commits on Jan 6, 2020` at the top of the commits tab but then the commit says its from 7 days ago...